client/src/helpers/APICalls/notifications.ts

```diff
@@ -0,0 +1,84 @@
+import { NotificationApiData } from '../../interface/Notifications';
+import { FetchOptions } from '../../interface/FetchOptions';
+
+export const createNotification = async (
+  type: string,
+  title: string,
+  description: string,
+): Promise<NotificationApiData> => {
+  const fetchOptions: FetchOptions = {
+    method: 'POST',
+    headers: { 'Content-Type': 'application/json' },
+    body: JSON.stringify({ type, title, description }),
+    credentials: 'include',
+  };
+
+  try {
+    const fetchData = await fetch(`/notification`, fetchOptions);
+    const notification = await fetchData.json();
+    return notification;
```

Comment:
Since this function returns a Promise, you can simply return `fetchData.json();` and await it in the calling function.

Reply:
@iamalx Please address comment above :)